.github/workflows/ci.yaml

```diff
@@ -12,20 +14,24 @@ jobs:
       matrix:
         node-version: [18]
     steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
This has to be run before `pnpm/action-setup` so that `package.json` is present on file system and version can be resolved from there.